src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -456,6 +456,12 @@ static constexpr Property<Level> level{"LOG_LEVEL"};
  */
 static constexpr Property<std::string> cache_dir{"CACHE_DIR"};
 
+/**
+ * @brief Read-only property to notify user that compiled model was loaded from the cache
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<bool, PropertyMutability::RO> from_cache{"FROM_CACHE"};
```

Comment:
maybe `loaded_from_cache` or `imported_from_cache` ?

Reply:
Ok, I will change to `loaded_from_cache`